.travis.yml

```diff
@@ -45,6 +47,7 @@ install:
   - gem install bundler
   - bundle install
   - git clone https://github.com/DataDog/dd-agent.git ~/dd-agent
+  - cp ~/dd-agent/datadog.conf.example ~/dd-agent/datadog.conf
```

Comment:
You won't need this after changes I introduced with the latest `datadog-sdk-testing` gem. See https://github.com/DataDog/integrations-core/pull/83. Once that's merged, just rebase to the latest `master`. That should do it.